dotcom-rendering/src/web/lib/renderElement.tsx

```diff
@@ -136,7 +136,7 @@ export const renderElement = ({
 }: Props): [boolean, JSX.Element] => {
 	const palette = decidePalette(format);
 
-	const isLiveBlog =
+	const isBlog =
```

Comment:
On the naming of this, what is a `blog`? Is that a live or dead blog? I thought the language was liveblog with live `status = true|false`? And if the status is false, we sometimes refer to it as deadblog. Never heard of just a plain blog.

Reply:
I guess a liveblog can be dead, but in format, you have `ArticleDesign.LiveBlog` and `ArticleDesign.DeadBlog`, so by extension and live-or-dead-blog could be considered just a blog? Unfortunately I've seen 'blog' refer to other things at the guardian as well!

Reply:
As this is co-located with the following `format.design` condition, it feels clear enough even for a newcomer that implies either a LiveBlog or a DeadBlog.

Reply:
I think there's also some history to this lexicon with references to 'sports blogs' and 'blogging'.